packages/larva-tokens/src/brands/indiewire.json

```diff
@@ -0,0 +1,108 @@
+{
+	"imports": [
+		"../base/all.json"
+	],
+	"props": {
+		"BACKGROUND_COLOR_BRAND_PRIMARY": {
+			"value": "#005DA4",
+			"type": "color",
+			"category": "background-color"
+		},
+		"BORDER_COLOR_BRAND_PRIMARY": {
+			"value": "#005DA4",
+			"type": "color",
+			"category": "hr-color"
+		},
+		"COLOR_BRAND_PRIMARY": {
+			"value": "#005DA4",
+			"type": "color",
+			"category": "text-color"
+		},
```

Comment:
@laras126 The style guide lists a`Primary` color. So I added three properties here for text, borders and backgrounds.

Does that seem right to you? (Should I remove `BACKGROUND_COLOR_BRAND_PRIMARY` and `BORDER_COLOR_BRAND_PRIMARY`?)

Reply:
Oh! And does it seem okay that the color values appear more than once? I suppose I could add it to `aliases.json` For example…
```
"INDIE_BLUE": "#005DA4",
```
```
"value": "{!INDIE_BLUE}"
```
Would that be much better, or no biggie?

Reply:
That is correct! It is a little verbose, but I think we should maintain that distinction in the future once we are able to think more intentionally about colors and the contexts where they are used (e.g. we might want a different hue for the border version of color-brand-primary).

Reply:
And I think no biggie for the alias - it can stay as a hex.